wordpress-mobile/WordPressKit-iOS
# Feature/invite links

Refs #https://github.com/wordpress-mobile/WordPress-iOS/issues/15356

This PR adds API support for invite links.  It includes: 
- Methods for fetching, generating, and disabling invite links
- A new model to represent remote invite links
- Basic unit tests for the new methods.

It's worth mentioning that the data returned from the v1.1 invites and v2.0 generate endpoints differs.  The new remote model relies on default values for fields missing from the generate endpoint. 

@ScoutHarris would you be game for a review? 

### Testing Details

To Test: 
Run unit tests.  Ensure they all pass.
Run the [companion WPiOS PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/15970) to confirm that the new methods work as expected.
